src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -768,8 +770,7 @@ private async Task ConfigureDeploymentFromCli(Recommendation recommendation, IEn
                 if (string.IsNullOrEmpty(input))
                 {
                     var validatorFailedResults =
-                        recommendation.Recipe

```

Comment:
same comment as above

Reply:
Updated both existing recipe validators with a `IOptionSettingHandler` in their constructor, which is injected in the new `ValidatorFactory`.  Removed `_optionSettingHandler` from the `Validate` calls.